channels/alpha

```diff
@@ -137,11 +137,11 @@ spec:
     requiredVersion: 1.11.10
   kopsVersions:
   - range: ">=1.25.0-alpha.1"
-    recommendedVersion: "1.25.0"
+    recommendedVersion: "1.25.2"
     #requiredVersion: 1.25.0
     kubernetesVersion: 1.25.3
   - range: ">=1.24.0-alpha.1"
-    recommendedVersion: "1.24.3"
+    recommendedVersion: "1.24.4"
```

Comment:
Time to default to 1.25.2 for older k8s versions too?
```suggestion
    recommendedVersion: "1.25.2"
```

Reply:
Yeah. Let's do both.
Not sure about the AMI given that another one was released today.

Reply:
Let's update AMI to latest.

Reply:
Updated AMIs for alpha.

Reply:
good catch. done.